project/pyproject.toml.jinja

```diff
@@ -88,9 +91,8 @@ help.cmd = "python src/main.py --help"
 
 aeroset.help = "Use a specific reference/tag/version of Aeromancy"
 aeroset.shell = """pdm remove aeromancy ;
-  pdm add 'git+ssh://git@github.com/quant-aq/aeromancy.git@{args}' &&
-  pdm install &&
-  pre-commit run keep-sorted --files pyproject.toml > /dev/null
+  pdm add 'git+https://github.com/quant-aq/aeromancy@{args}' &&
+  pdm install
```

Comment:
it's not the point of this change, but `pdm update ...` might combine some of these commands (and run faster)